server/routerlicious/packages/lambdas-driver/src/kafka-service/partitionManager.ts

```diff
@@ -169,10 +169,6 @@ export class PartitionManager extends EventEmitter {
                     return;
                 }
 
-                // For simplicity we will close the entire manager whenever any partition errors. In the case that the
-                // restart flag is false and there was an error we will eventually need a way to signify that a
-                // partition is 'poisoned'.
-                errorData.restart = true;
```

Comment:
Hey @tanviraumi this line of code was overriding the restart value, causing Scribe to restart indefinitely even after the fixes in this PR. I wanted to check with you whether this can be removed - @GaryWilber and I agree that we should not be overriding the error data upstream, leaving it to whoever throws the error to decide whether to restart or not.

Reply:
Sounds good to me.